core/policy.go

```diff
@@ -15,7 +15,7 @@ import (
 // pools a little before the sending node gives up on them.
 const OutboxMaxAgeRounds = 10
 
-var log = logging.Logger("mqueue")
+var log = logging.Logger("core")
```

Comment:
This logger is shared by both inbound and outbound message handling, so the previous name was misleading and its location in `policy.go` arbitrary. We can do better when the `core` package is reorganized but I don't want to shave that yak today.